[2022-09-05 19:56:09,845] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: my_dag_postgres2.create_postgre_table scheduled__2021-01-01T00:00:00+00:00 [queued]>
[2022-09-05 19:56:09,870] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: my_dag_postgres2.create_postgre_table scheduled__2021-01-01T00:00:00+00:00 [queued]>
[2022-09-05 19:56:09,873] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 19:56:09,878] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-09-05 19:56:09,882] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-09-05 19:56:09,905] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_postgre_table> on 2021-01-01 00:00:00+00:00
[2022-09-05 19:56:09,915] {standard_task_runner.py:52} INFO - Started process 2243 to run task
[2022-09-05 19:56:09,920] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag_postgres2', 'create_postgre_table', 'scheduled__2021-01-01T00:00:00+00:00', '--job-id', '3821', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmp6ruyaec6', '--error-file', '/tmp/tmpsf9r05us']
[2022-09-05 19:56:09,921] {standard_task_runner.py:80} INFO - Job 3821: Subtask create_postgre_table
[2022-09-05 19:56:10,063] {task_command.py:371} INFO - Running <TaskInstance: my_dag_postgres2.create_postgre_table scheduled__2021-01-01T00:00:00+00:00 [running]> on host eba23da77ca5
[2022-09-05 19:56:10,184] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag_postgres2
AIRFLOW_CTX_TASK_ID=create_postgre_table
AIRFLOW_CTX_EXECUTION_DATE=2021-01-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-01-01T00:00:00+00:00
[2022-09-05 19:56:10,207] {base.py:68} INFO - Using connection ID 'postgres_localhost' for task execution.
[2022-09-05 19:56:10,240] {dbapi.py:231} INFO - Running statement: 
        create table if not exists dag_runs (
            dt date,
            dag_id character varying,
            primary key (dt,dag_id)
        ) 
        , parameters: None
[2022-09-05 19:56:10,243] {postgres.py:94} INFO - NOTICE:  relation "dag_runs" already exists, skipping

[2022-09-05 19:56:10,269] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=my_dag_postgres2, task_id=create_postgre_table, execution_date=20210101T000000, start_date=20220905T195609, end_date=20220905T195610
[2022-09-05 19:56:10,336] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-05 19:56:10,408] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
